cmd/util/cmd/export-json-transactions/cmd.go

```diff
@@ -0,0 +1,122 @@
+package export_transaction
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-go/cmd/util/cmd/common"
+	"github.com/onflow/flow-go/cmd/util/cmd/export-json-transactions/transactions"
+)
+
+var flagDatadir string
+var flagOutputDir string
+var flagStartHeight uint64
+var flagEndHeight uint64
+
+// example:
+// ./util export-json-transactions --output-dir ./ --datadir /var/fow/data/protocol/ --start-height 2 --end-height 242
+var Cmd = &cobra.Command{
+	Use:   "export-json-transactions",
+	Short: "exports transactions into a json file",
+	Run:   run,
+}
+
+func init() {
+	Cmd.Flags().StringVar(&flagDatadir, "datadir", "/var/flow/data/protocol",
+		"the protocol state")
+
+	Cmd.Flags().StringVar(&flagOutputDir, "output-dir", "",
+		"Directory to write transactions JSON to")
+	_ = Cmd.MarkFlagRequired("output-dir")
+
+	Cmd.Flags().Uint64Var(&flagStartHeight, "start-height", 0,
+		"Start height of the block range")
+	_ = Cmd.MarkFlagRequired("start-height")
+
+	Cmd.Flags().Uint64Var(&flagEndHeight, "end-height", 0,
+		"End height of the block range")
+	_ = Cmd.MarkFlagRequired("end-height")
+}
+
+func run(*cobra.Command, []string) {
+	log.Info().Msg("start exporting transactions")
+	err := ExportTransactions(flagDatadir, flagOutputDir, flagStartHeight, flagEndHeight)
+	if err != nil {
+		log.Fatal().Err(err).Msg("cannot get export transactions")
+	}
+	log.Info().Msg("transactions exported")
+}
+
+func writeJSONTo(writer io.Writer, jsonData []byte) error {
```

Comment:
This looks like an re-implementation of the [os.WriteFile](https://pkg.go.dev/os#WriteFile)

Reply:
Yeah, I was thinking to keep the option of using Writer for pipelining the block processing